app/assets/javascripts/dashboard.js

```diff
@@ -1,32 +1,57 @@
-//= require leaflet.locate
-
 $(function () {
   const defaultHomeZoom = 12;
   let map;
 
   if ($("#map").length) {
-    map = L.map("map", {
+    map = new maplibregl.Map({
+      container: "map",
+      style: {
+        version: 8,
+        name: "OSM Raster",
+        sources: {
+          osm: {
+            type: "raster",
+            tiles: [
+              "https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+            ],
+            tileSize: 256,
+            maxzoom: 19,
+            attribution: "© OpenStreetMap contributors"
+          }
+        },
+        layers: [
+          {
+            id: "osm",
+            type: "raster",
+            source: "osm"
+          }
+        ]
+      },
```

Comment:
Long-term, we’ll probably want a system for centralizing style configuration like we have for layers on the main slippy map:

https://github.com/openstreetmap/openstreetmap-website/blob/9ff2103e60832de0f45c993fda22e4ca51e5303b/app/assets/javascripts/leaflet.maptiler.js#L4